README.md

```diff
@@ -1,5 +1,9 @@
 # HR-TIL
 
+![til](https://raw.githubusercontent.com/michaeldimmitt/hr-til/readme-oauth-instructions/app/assets/images/banner.png)
+
+<!--![til](https://raw.githubusercontent.com/hashrocket/hr-til/master/app/assets/images/banner.png)-->
```

Comment:
@jwworth, made the change requested for the image on the readme. Currently showing the image on the branch  that I am making the pull request so that you see that it is working. 

However, before you accept the pull request you can swap line three with line 5.